docs/course_materials/exercises/01_Linux/check_homework_1.py

```diff
@@ -1,27 +1,15 @@
 from pathlib import Path
 import check_helper
 
-exercise_commands = {
-    'pwd': 'printing current directory name',
-    'mkdir': 'creating directories',
-    'cd': 'changing between directories',
-    'ls': 'listing directory contents',
-    'tree': 'viewing the directory tree',
-    'rm': 'removing a file or directory',
-    'touch': 'creating new files',
-    'cat': 'printing the contents of a file',
-    'vim': 'for editing a file',
-    'website.txt': 'writing your favorite song (partial command)', 
-    'docs': 'a folder in hw1 (partial command)'
-    } 
+exercise_commands = ['pwd', 'mkdir', 'cd', 'ls', 'tree', 'rm', 'touch', 'cat', 'vim', 'website.txt', 'docs'] 
 
-exercise_paths = [check_helper.HOME + '/hw1',
-                check_helper.HOME + '/hw1/web',
-                check_helper.HOME + '/hw1/other',
+exercise_paths = ['/hw1',
+                '/hw1/web',
+                '/hw1/other',
```

Comment:
I would also remove from here the first / and add it in the function .
because these are a relative path.